docs/source/conf.py

```diff
@@ -27,11 +27,12 @@
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 sys.path.append(os.path.join(PATH_RAW_NB, ".actions"))
 
-_SHOULD_COPY_NOTEBOOKS = True
+_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("_SHOULD_COPY_NOTEBOOKS", 1)))
```

Comment:
lets keep this as normal
```suggestion
_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("SHOULD_COPY_PL_NOTEBOOKS", 1)))
```

Reply:
Done!